workflow/app.py

```diff
@@ -745,17 +765,27 @@ def flightline_map(line, vmin, vmax, layer):
 def update_interp_table(clickData, previous_table, section, section_tab, line, vmin, vmax, current_table,
                         surfaceName, interpreted_points, model, pmap_store):
     trig_id = find_trigger()
+
+    #a bunch of "do nothing" cases here - before doing any data transformations to save
+    #on useless computation
+    if trig_id == 'section_plot.clickData' and section_tab == 'data_section':
+        #clicked on the data section so nothing to do
+        raise PreventUpdate
+
     # Access the data from the store. The or is in case of None callback at initialisation
     interpreted_points = interpreted_points or df_interpreted_points.to_dict('records')
     pmap_store = pmap_store or {}
     df = pd.DataFrame(interpreted_points).infer_objects()
     if section == "lci":
+        #file system access every time the user clicks a point is bound to be very slow
+        #TODO don't do this - not sure how to set global state for xarr 
```

Comment:
This is actually somewhat of a technical challenge unless one sets a global variable.